x-pack/plugins/actions/server/init.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Legacy } from 'kibana';
+import { ActionService } from './action_service';
+import { ActionTypeService } from './action_type_service';
+import {
+  createRoute,
+  deleteRoute,
+  findRoute,
+  getRoute,
+  updateRoute,
+  listActionTypesRoute,
+} from './routes';
+
+export function init(server: Legacy.Server) {
```

Comment:
Is there anything missing here for a plugin? `xpackMainPlugin.registerFeature`, rbac, etc

cc @kobelb

Reply:
You'll only want to register a feature if you wish for it to show up as a discrete piece of functionality that a user can enable/disable using spaces disabled features or role's feature privileges. I'm thinking we'd likely have "Alerting" itself as the feature, and this would be implicitly enabled/disabled.

Reply:
Thanks for the info, I will wait until we have an "Alerting" app before registering a feature.